### ISSUE TITLE
Thoughts on ReactMini?

### ISSUE THREAD

**Issue by phated** [lpalmes/pure#4]
This project seems super cool and actually aligns with an [idea I had back in Oct](https://twitter.com/BlaineBublitz/status/1055910941290725377).

What are your thoughts on [ReactMini](https://github.com/reasonml/reason-react/tree/master/ReactMini)?  I was thinking about using that as the reconciler (if I'm using that term correctly), but it seems like your have implemented your own?

**Comment by lpalmes** [comment]
I really like ReactMini, it was a first introduction (for me) to a pure react Reconciler in Reason, i would love swapping the reconciler between this fiber-like Reconciler with the ReactMini one. We even can accept multiple reconcilers that satisfy a reconciler spec.

I like that the ReactMini is immutable, that would be quite a benefit for a lot of ideas, i don't know how we could do hot code reloading like reprocessing given that it uses a small trick to evade Obj.magic, and i abuse Obj.magic in my reconciler to make [hot code reloading happen](https://www.youtube.com/watch?v=7o_7eUIPPko). 

 I'm starting to spec how we can accept native components and including third party components, and i'm finishing a migration to esy right now.

TLDR: i really like the ReactMini reconciler and i would like to try it out here
